workflow-service/src/main/java/com/redhat/parodos/project/controller/ProjectController.java

```diff
@@ -53,6 +54,7 @@
 
 @CrossOrigin(origins = "*", maxAge = 1800)
 @RestController
+@Validated
```

Comment:
Do you think the test should be updated?

Reply:
yes, but pls let me add them as integration tests, after your PR is merged.